test/consumer.test.ts

```diff
@@ -41,7 +41,7 @@ describe('Response consumer test', () => {
                     ]
                 }
             ],
-            "message": "abc - 1.2.3 has 1 known security vulnerability and 1 security advisory with 1 having critical severity. Recommendation: use version 2.3.4. ",
+            "message": "abc: 1.2.3\nknown security vulnerability: 1\nsecurity advisory: 1\nhighest severity: critical\nRecommendation: use version 2.3.4.",
```

Comment:
You no longer use message from the response right? If so, why the test response has been modified.?

Reply:
Ohh, My mistake.

Thanks!